eahub/base/static/scripts/maps/heatmap.js

```diff
@@ -146,13 +147,20 @@ function addDescription(marker, profiles) {
       marker.desc += '<li>' + private_profiles.length.toString() + ' anonymous ' + user_word + '</li>'
     }
     marker.desc += '</ul>'
-  } else if (public_profiles.length == 1) {
+  } else if (!exists(public_profiles) && exists(private_profiles)) {
```

Comment:
You don't need an `exists` function, this can just be `} else if (!public_profiles.length && private_profiles.length) {`.

Reply:
I just thought it reads easier